### PR TITLE
mask refinement: fix opencl codepath for bug in opacity/contrast handling

### PR DESCRIPTION
**Pull request by upegelow**
fix for #3629

### CODE REVIEW

File: data/kernels/blendop.cl [hunk 1]
```diff
@@ -1017,10 +1017,10 @@ blendop_mask_tone_curve(__read_only image2d_t mask_in, __write_only image2d_t ma
   const int x = get_global_id(0);
   const int y = get_global_id(1);
 
-  if ( x>= width || y >= height || gopacity <= 1e-4f) return;
+  if (x >= width || y >= height) return;
 
   float opacity = read_imagef(mask_in, sampleri, (int2)(x, y)).x;
-  float scaled_opacity = (2.f * opacity - 1.f) / gopacity;
+  float scaled_opacity = (2.f * opacity / gopacity - 1.f);
```
Comment (TurboGit) [inline data/kernels/blendop.cl]: So we can have a negative scaled_opacity ?

Comment (TurboGit) [reply data/kernels/blendop.cl]: Looking add the line below, it seems that yes. Strange but just want to be sure.

Comment (upegelow) [reply data/kernels/blendop.cl]: > So we can have a negative scaled_opacity ?

Don't really understand that code. This PR is just about making CPU == GPU.

**Comment by groutr** [comment]
I verified that this does indeed fix #3629.

**Pull request merged**